### ISSUE TITLE
the view from the mobile does not work

### ISSUE THREAD

**Issue by Diegopintos** [select/audius#63]
Why when I access from my mobile device do my lists not show?

In the browser, it works perfect, but on mobile devices it does not work

**Comment by select** [comment]
Yeah, I was doing some major rework of the client in version 2.0.12 and broke the mobile interface in the process. I'm currently in the process of making it work again and reorganizing it so you can also use all new features in the mobile client.

Could you tell me what exactly is missinge, is it the playlists tab content or the rooms tab content?

**Comment by Diegopintos** [comment]
Everything looks perfectly ... just that it does not save the tracks and lists that I had created.
It looks completely like factory.

**Comment by select** [comment]
Thanks that helps, I think this weekend I will have some time again to work on it.
